src/olympia/scanners/tests/test_admin.py

```diff
@@ -39,7 +39,7 @@ def setUp(self):
         super().setUp()
 
         self.user = user_factory()
-        self.grant_permission(self.user, 'Admin:Advanced')
+        self.grant_permission(self.user, 'Admin:*')
```

Comment:
That's because most test cases assume the current user is an admin. It was working before because we only one permission but now we have more permissions. Admins have `Admin:*` (or rather `*:*`) so that should not change anything for them.

FWIW, this would be a problem for users with `Admin:Advanced` only (but I don't think we have that) and admins would need to give `Admin:ScannersResultsView`, etc. to these users.